CHANGELOG.md

```diff
@@ -4,6 +4,60 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [0.38.0] - 2024-10-24
+
+This release doesn't introduce any substantial breaking changes and focuses primarily on incremental improvements, testing and bug fixes. A few of the highlights include:
```

Comment:
do want to say something about v5 bare extrinsics decoding support?

or does that patch release 0.37.1 cover that?

Reply:
I mentioned this as the top highlight below if that's what you have in mind? :)

Reply:
ah cool missed that :(